tensorflow/tensorflow.bzl

```
@@ -362,9 +362,12 @@ def tf_copts(
         if_libtpu(["-DLIBTPU_ON_GCE"], []) +
         if_xla_available(["-DTENSORFLOW_USE_XLA=1"]) +
         if_tensorrt(["-DGOOGLE_TENSORRT=1"]) +
-        if_mkl(["-DINTEL_MKL=1"]) +
-        if_mkldnn_openmp(["-DENABLE_ONEDNN_OPENMP"]) +
+        # Compile in oneDNN based ops when building for x86 platforms
+        if_mkl(["-DINTEL_MKL"])+
+        # Enable additional ops and optimizations for Intel builds using
```

```suggestion
        # Enable additional ops (e.g., ops with non-NHWC data layout) and optimizations for Intel builds using
```
(Please also re-wrap the line.)

---

Will make code change per suggestion